CONTRIBUTION.md

```diff
@@ -118,19 +118,20 @@ storiesOf('Molecules/Top content bar', module)
 
 ## How-to release a component
 
+**Do not use the release scripts: ~~`npm run release`~~ or ~~`npm run prerelease`~~**
```

Comment:
So, lets delete these scripts....!

Reply:
Usually I would agree, but AFAIK luca is having a plan here (and already uses them).